subsys/bluetooth/audio/Kconfig.tbs

```diff
@@ -170,6 +170,158 @@ config BT_TBS_CLIENT_MAX_TBS_INSTANCES
 	  Sets the maximum number of Telephone Bearer Service (TBS)
 	  instances to setup and use.
 
+config BT_TBS_CLIENT_BEARER_PROVIDER_NAME
+	bool "Read Bearer Provider Name Procedure"
+	default y
+	help
+	  This option enables support for the Read Bearer Provider Name
+	  procedure.
```

Comment:
I do wonder if we should just write something like

```
config BT_TBS_CLIENT_BEARER_PROVIDER_NAME
	bool "Support reading Bearer Provider Name"
	default y
	help
	  This option enables support for reading the Bearer Provider Name.
```

This applies to most of the Kconfigs if we change it 

What do you think?

Reply:
The proposed change is shorter, and probably more understandable to the user.  So I am in favour of it.